website/site/content/docs/configuration-options.md

```diff
@@ -184,3 +184,13 @@ This setting changes options for the editor view of the collection. It has one o
   editor:
      preview: false
 ```
+
+### `display_url`
```

Comment:
This key isn't a per-collection setting, so it should probably go in a general section, maybe near "Media and Public Folders".

Reply:
I agree. This also means that the heading would change to an H2, and I would format it "Display URL".

Reply:
Gotcha, I see what you mean. Changed. 👍